.github/pull_request_template

```diff
@@ -6,5 +6,6 @@ Explain your rationale of technical decisions you made (unless discussed before)
 Add links to github/jira issues, design documents and other relevant resources (e.g. previous discussions, platform/tool documentation etc.)
 
 # Checklist
-- [ ] I wrote/updated tests for new/changed code
-- [ ] I ran `npm run build` without issues
\ No newline at end of file
+- [ ] I wrote/updated tests for new/changed code.
+- [ ] I ran `yarn sdk buildSdk` `yarn bridge buildBridge` without issues.
+- [ ] I ran `yarn sample prettify` `yarn sample build` without issues.
```

Comment:
Probably this is okay for now, but maybe we can add a script in the root `package.json` that will run the checks in all of the workspaces, and then this will be much easier.

Reply:
Yeah, put all those commands into root `package.json` which get errors eslint. Looks like we need to config it later.